src/main/kotlin/com/example/pass_manager/domain/Client.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.pass_manager.domain
+
+import com.example.pass_manager.domain.Client.Companion.COLLECTION_NAME
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.TypeAlias
+import org.springframework.data.mongodb.core.mapping.Document
+
+@TypeAlias("MongoClient")
+@Document(collection = COLLECTION_NAME)
+data class Client(
```

Comment:
`Client` is too generic word. I have no idea what it's about. Let's rename to something more clear